### ISSUE TITLE
Incorrect batch size info with keras.utils.Sequence

### ISSUE THREAD

**Issue by sciann** [tensorflow/tensorflow#48788]
TensorFlow 2.4.1

Batch size info is miscalculated when data is wrapped in Sequence. The example below is the output for the first epoch training on 10,000 points with batch_size=32, which should result in 313 mini-batch updates.  The batch number on the far left is correct (313/313), however, the size and batch before loss seem to be incorrect. 

Output: 
Epoch 1/100
313/313 [==============================] - 1s 966us/step - **_batch: 156.0000 - size: 31.9489_** - loss: 0.1579

**Comment by UsharaniPagadala** [comment]
@sciann 
Could you please fill the template.
and also share more details for us to analyse, simple stand alone code to replicate the issue faced or a colab gist with the code and error.Thanks

**Comment by sciann** [comment]
I had a chance to debug the issue, and it seems to be rooted in the `train_generator_v1.py`. 

Please use `tf.compat.v1.disable_eager_execution()` at the beginning of the code to get the same problem. 

SciANN works on the TF1.
